troubadix/plugins/vt_placement.py

```diff
@@ -25,6 +25,8 @@
 from troubadix.helper.patterns import _get_special_script_tag_pattern
 from troubadix.plugin import FileContentPlugin, LinterError, LinterResult
 
+FEED_VERSIONS = ["common", "21.04", "22.04", ""]
```

Comment:
As this is also in use in dependencies.py since #271 wouldn't it make sense to somehow define this more globally so that it needs to be only updated at one place once there is a new GVM version?

Reply:
@pascalholthaus he is right. Maybe add this into https://github.com/greenbone/troubadix/blob/main/troubadix/helper/helper.py, we already defined other statics there. :)

Reply:
Done